soc/xtensa/intel_adsp/ace_v1x/soc.c

```diff
@@ -45,13 +45,13 @@ static __imr void hp_sram_pm_banks(void)
 	uint32_t hpsram_ebb_quantity = mtl_hpsram_get_bank_count();
 	volatile uint32_t *l2hsbpmptr = (volatile uint32_t *)MTL_L2MM->l2hsbpmptr;
 	volatile uint8_t *status = (volatile uint8_t *)l2hsbpmptr + 4;
-	int inx;
+	int idx;
```

Comment:
Minor item: **idx** should be of type **uint32_t** to match **hpsram_ebb_quantity**